integration_tests/androidx_test/src/test/java/org/robolectric/integrationtests/axt/EspressoTest.java

```diff
@@ -115,6 +116,25 @@ public void textView() {
               assertThat(view.getHeight()).isGreaterThan(0);
             });
     onView(withText("Text View")).check(matches(isCompletelyDisplayed()));
+    onView(withText("Text View"))
```

Comment:
If you add a new test method called `textViewWithNegativeScaleX`, what about moving this part to a new method called `textViewWithPositiveScaleX`, and one called `testViewWithLetterSpacing`?

Reply:
Fixed.
I added one view for one attribute and one test for each view. Now the tests look more independent of each other.